msu/hooks/entity/tactical/actor.nut

```diff
@@ -30,6 +32,17 @@
 		return ret;
 	}
 
+	local onSkillsUpdated = o.onSkillsUpdated;
+	o.onSkillsUpdated = function()
+	{
+		onSkillsUpdated();
+		if (!this.m.IsDying)
+		{
+			this.m.MSU_callOnSkillsUpdated = !this.m.MSU_callOnSkillsUpdated;
+			if (this.m.MSU_callOnSkillsUpdated) this.getSkills().onSkillsUpdated();
```

Comment:
Intuitively, the name is backwards, we don't really name variables in a way that tells us what to do.

Reply:
Check now and lemme know if you are ok with the change.